canvasapi/assignment.py

```diff
@@ -254,6 +330,27 @@ def set_extensions(self, assignment_extensions, **kwargs):
             for extension in extension_list
         ]
 
+    def show_provisonal_grades_for_student(self, **kwargs):
+        """
+        :call: GET /api/v1/courses/:course_id/assignments/:assignment_id/
+        anonymous_provisional_grades/status \
+        <https://canvas.instructure.com/doc/api/all_resources.html#method.anonymous_provisional_grades.status>
+        :param user: The user that will be used
+        :type user: :class:`canvasapi.user.User`
```

Comment:
Similar concern as my comments on the `user` docstrings on `get_provisional_grades_status`.